[2024-03-09T02:45:18.259+0000] {processor.py:161} INFO - Started process (PID=1085) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:45:18.260+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T02:45:18.261+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:45:18.261+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:45:18.271+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:45:18.281+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.024 seconds
[2024-03-09T02:45:28.319+0000] {processor.py:161} INFO - Started process (PID=1086) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:45:28.320+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T02:45:28.320+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:45:28.320+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:45:28.343+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:45:28.338+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T02:45:28.344+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:45:28.358+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.041 seconds
[2024-03-09T02:45:47.929+0000] {processor.py:161} INFO - Started process (PID=1095) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:45:47.930+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T02:45:47.930+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:45:47.930+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:45:47.950+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:45:47.946+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T02:45:47.952+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:45:47.967+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.040 seconds
[2024-03-09T02:45:58.034+0000] {processor.py:161} INFO - Started process (PID=1096) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:45:58.035+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T02:45:58.036+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:45:58.036+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:45:58.056+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:45:58.052+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T02:45:58.057+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:45:58.073+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.042 seconds
[2024-03-09T02:46:28.220+0000] {processor.py:161} INFO - Started process (PID=1105) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:46:28.221+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T02:46:28.222+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:46:28.221+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:46:28.234+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:46:28.231+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T02:46:28.235+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:46:28.249+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.032 seconds
[2024-03-09T02:46:36.314+0000] {processor.py:161} INFO - Started process (PID=1114) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:46:36.315+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T02:46:36.316+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:46:36.316+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:46:36.336+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:46:36.332+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T02:46:36.337+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:46:36.351+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.040 seconds
[2024-03-09T02:47:02.443+0000] {processor.py:161} INFO - Started process (PID=1115) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:47:02.443+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T02:47:02.444+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:47:02.444+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:47:02.461+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:47:02.457+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T02:47:02.462+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:47:02.477+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.037 seconds
[2024-03-09T02:47:30.943+0000] {processor.py:161} INFO - Started process (PID=1124) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:47:30.944+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T02:47:30.945+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:47:30.945+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:47:30.953+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:47:30.952+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 22
    job_task =
              ^
SyntaxError: invalid syntax
[2024-03-09T02:47:30.953+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:47:30.968+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.027 seconds
[2024-03-09T02:48:01.117+0000] {processor.py:161} INFO - Started process (PID=1133) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:48:01.118+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T02:48:01.119+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:48:01.118+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:48:01.127+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:48:01.126+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 22
    job_task =
              ^
SyntaxError: invalid syntax
[2024-03-09T02:48:01.127+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:48:01.143+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.029 seconds
[2024-03-09T02:48:31.292+0000] {processor.py:161} INFO - Started process (PID=1142) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:48:31.292+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T02:48:31.293+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:48:31.293+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:48:31.302+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:48:31.301+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 839, in exec_module
  File "<frozen importlib._bootstrap_external>", line 976, in get_code
  File "<frozen importlib._bootstrap_external>", line 906, in source_to_code
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 22
    job_task =
              ^
SyntaxError: invalid syntax
[2024-03-09T02:48:31.302+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:48:31.317+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.028 seconds
[2024-03-09T02:48:53.445+0000] {processor.py:161} INFO - Started process (PID=1151) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:48:53.446+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T02:48:53.446+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:48:53.446+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:48:53.472+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:48:53.467+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T02:48:53.473+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:48:53.491+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.048 seconds
[2024-03-09T02:49:23.638+0000] {processor.py:161} INFO - Started process (PID=1160) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:49:23.639+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T02:49:23.639+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:49:23.639+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:49:23.652+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:49:23.649+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T02:49:23.653+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:49:23.668+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.032 seconds
[2024-03-09T02:49:53.808+0000] {processor.py:161} INFO - Started process (PID=1169) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:49:53.809+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T02:49:53.810+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:49:53.809+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:49:53.823+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:49:53.819+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T02:49:53.824+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:49:53.839+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.033 seconds
[2024-03-09T02:50:14.288+0000] {processor.py:161} INFO - Started process (PID=1178) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:50:14.289+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T02:50:14.290+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:50:14.290+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:50:14.312+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:50:14.308+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T02:50:14.314+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:50:14.333+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.047 seconds
[2024-03-09T02:50:44.487+0000] {processor.py:161} INFO - Started process (PID=1187) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:50:44.488+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T02:50:44.488+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:50:44.488+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:50:44.503+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:50:44.499+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T02:50:44.504+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:50:44.520+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.036 seconds
[2024-03-09T02:51:14.652+0000] {processor.py:161} INFO - Started process (PID=1196) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:51:14.653+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T02:51:14.653+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:51:14.653+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:51:14.666+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:51:14.663+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T02:51:14.667+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:51:14.682+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.033 seconds
[2024-03-09T02:51:44.812+0000] {processor.py:161} INFO - Started process (PID=1205) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:51:44.813+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T02:51:44.814+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:51:44.814+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:51:44.827+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:51:44.824+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T02:51:44.829+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:51:44.843+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.032 seconds
[2024-03-09T02:52:14.977+0000] {processor.py:161} INFO - Started process (PID=1214) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:52:14.978+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T02:52:14.979+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:52:14.979+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:52:14.992+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:52:14.989+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T02:52:14.993+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:52:15.009+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.034 seconds
[2024-03-09T02:52:41.152+0000] {processor.py:161} INFO - Started process (PID=1223) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:52:41.153+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T02:52:41.154+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:52:41.154+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:52:41.173+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:52:41.169+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T02:52:41.174+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:52:41.189+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.040 seconds
[2024-03-09T02:53:11.327+0000] {processor.py:161} INFO - Started process (PID=1232) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:53:11.328+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T02:53:11.328+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:53:11.328+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:53:11.341+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:53:11.338+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T02:53:11.342+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:53:11.358+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.033 seconds
[2024-03-09T02:53:41.503+0000] {processor.py:161} INFO - Started process (PID=1241) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:53:41.504+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T02:53:41.504+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:53:41.504+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:53:41.517+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:53:41.514+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T02:53:41.518+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:53:41.532+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.032 seconds
[2024-03-09T02:54:11.674+0000] {processor.py:161} INFO - Started process (PID=1250) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:54:11.675+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T02:54:11.676+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:54:11.676+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:54:11.690+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:54:11.687+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T02:54:11.691+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:54:11.708+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.036 seconds
[2024-03-09T02:54:41.865+0000] {processor.py:161} INFO - Started process (PID=1259) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:54:41.866+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T02:54:41.867+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:54:41.867+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:54:41.887+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:54:41.881+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T02:54:41.888+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:54:41.909+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.047 seconds
[2024-03-09T02:55:12.052+0000] {processor.py:161} INFO - Started process (PID=1268) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:55:12.053+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T02:55:12.053+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:55:12.053+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:55:12.068+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:55:12.065+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T02:55:12.069+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:55:12.088+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.039 seconds
[2024-03-09T02:55:42.254+0000] {processor.py:161} INFO - Started process (PID=1277) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:55:42.254+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T02:55:42.255+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:55:42.255+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:55:42.269+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:55:42.265+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T02:55:42.270+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:55:42.285+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.034 seconds
[2024-03-09T02:56:12.423+0000] {processor.py:161} INFO - Started process (PID=1286) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:56:12.424+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T02:56:12.425+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:56:12.425+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:56:12.439+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:56:12.435+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T02:56:12.441+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:56:12.456+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.035 seconds
[2024-03-09T02:56:42.595+0000] {processor.py:161} INFO - Started process (PID=1295) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:56:42.595+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T02:56:42.597+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:56:42.596+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:56:42.612+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:56:42.608+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T02:56:42.613+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:56:42.628+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.035 seconds
[2024-03-09T02:57:12.765+0000] {processor.py:161} INFO - Started process (PID=1304) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:57:12.766+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T02:57:12.766+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:57:12.766+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:57:12.780+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:57:12.777+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T02:57:12.781+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:57:12.795+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.033 seconds
[2024-03-09T02:57:42.925+0000] {processor.py:161} INFO - Started process (PID=1313) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:57:42.926+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T02:57:42.927+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:57:42.926+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:57:42.940+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:57:42.937+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T02:57:42.941+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:57:42.956+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.033 seconds
[2024-03-09T02:58:13.089+0000] {processor.py:161} INFO - Started process (PID=1322) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:58:13.090+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T02:58:13.091+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:58:13.091+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:58:13.103+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:58:13.100+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T02:58:13.104+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:58:13.119+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.032 seconds
[2024-03-09T02:58:43.245+0000] {processor.py:161} INFO - Started process (PID=1331) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:58:43.246+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T02:58:43.246+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:58:43.246+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:58:43.260+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:58:43.257+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T02:58:43.262+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:58:43.278+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.036 seconds
[2024-03-09T02:59:13.422+0000] {processor.py:161} INFO - Started process (PID=1340) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:59:13.423+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T02:59:13.423+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:59:13.423+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:59:13.436+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:59:13.433+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T02:59:13.437+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:59:13.454+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.034 seconds
[2024-03-09T02:59:43.580+0000] {processor.py:161} INFO - Started process (PID=1349) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:59:43.581+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T02:59:43.582+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:59:43.581+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:59:43.594+0000] {logging_mixin.py:188} INFO - [2024-03-09T02:59:43.591+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T02:59:43.595+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T02:59:43.609+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.031 seconds
[2024-03-09T03:00:13.729+0000] {processor.py:161} INFO - Started process (PID=1356) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:00:13.730+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T03:00:13.730+0000] {logging_mixin.py:188} INFO - [2024-03-09T03:00:13.730+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:00:13.744+0000] {logging_mixin.py:188} INFO - [2024-03-09T03:00:13.741+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T03:00:13.745+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:00:13.759+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.034 seconds
[2024-03-09T03:00:43.910+0000] {processor.py:161} INFO - Started process (PID=1365) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:00:43.910+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T03:00:43.911+0000] {logging_mixin.py:188} INFO - [2024-03-09T03:00:43.911+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:00:43.925+0000] {logging_mixin.py:188} INFO - [2024-03-09T03:00:43.921+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T03:00:43.926+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:00:43.941+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.033 seconds
[2024-03-09T03:01:14.101+0000] {processor.py:161} INFO - Started process (PID=1374) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:01:14.102+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T03:01:14.103+0000] {logging_mixin.py:188} INFO - [2024-03-09T03:01:14.103+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:01:14.119+0000] {logging_mixin.py:188} INFO - [2024-03-09T03:01:14.114+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T03:01:14.120+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:01:14.139+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.041 seconds
[2024-03-09T03:01:44.290+0000] {processor.py:161} INFO - Started process (PID=1383) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:01:44.290+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T03:01:44.291+0000] {logging_mixin.py:188} INFO - [2024-03-09T03:01:44.291+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:01:44.303+0000] {logging_mixin.py:188} INFO - [2024-03-09T03:01:44.300+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T03:01:44.304+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:01:44.317+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.030 seconds
[2024-03-09T03:02:14.466+0000] {processor.py:161} INFO - Started process (PID=1392) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:02:14.467+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T03:02:14.468+0000] {logging_mixin.py:188} INFO - [2024-03-09T03:02:14.468+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:02:14.481+0000] {logging_mixin.py:188} INFO - [2024-03-09T03:02:14.477+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T03:02:14.482+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:02:14.496+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.032 seconds
[2024-03-09T03:02:44.650+0000] {processor.py:161} INFO - Started process (PID=1401) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:02:44.651+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T03:02:44.653+0000] {logging_mixin.py:188} INFO - [2024-03-09T03:02:44.652+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:02:44.668+0000] {logging_mixin.py:188} INFO - [2024-03-09T03:02:44.664+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T03:02:44.669+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:02:44.685+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.038 seconds
[2024-03-09T03:03:14.834+0000] {processor.py:161} INFO - Started process (PID=1410) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:03:14.834+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T03:03:14.835+0000] {logging_mixin.py:188} INFO - [2024-03-09T03:03:14.835+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:03:14.847+0000] {logging_mixin.py:188} INFO - [2024-03-09T03:03:14.844+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T03:03:14.848+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:03:14.864+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.033 seconds
[2024-03-09T03:03:45.015+0000] {processor.py:161} INFO - Started process (PID=1419) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:03:45.016+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T03:03:45.016+0000] {logging_mixin.py:188} INFO - [2024-03-09T03:03:45.016+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:03:45.029+0000] {logging_mixin.py:188} INFO - [2024-03-09T03:03:45.026+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T03:03:45.030+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:03:45.045+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.033 seconds
[2024-03-09T03:04:15.190+0000] {processor.py:161} INFO - Started process (PID=1428) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:04:15.191+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T03:04:15.192+0000] {logging_mixin.py:188} INFO - [2024-03-09T03:04:15.192+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:04:15.203+0000] {logging_mixin.py:188} INFO - [2024-03-09T03:04:15.201+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T03:04:15.204+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:04:15.218+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.030 seconds
[2024-03-09T03:04:45.368+0000] {processor.py:161} INFO - Started process (PID=1437) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:04:45.369+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T03:04:45.370+0000] {logging_mixin.py:188} INFO - [2024-03-09T03:04:45.369+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:04:45.382+0000] {logging_mixin.py:188} INFO - [2024-03-09T03:04:45.379+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T03:04:45.383+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:04:45.397+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.032 seconds
[2024-03-09T03:05:15.543+0000] {processor.py:161} INFO - Started process (PID=1446) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:05:15.544+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T03:05:15.545+0000] {logging_mixin.py:188} INFO - [2024-03-09T03:05:15.545+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:05:15.558+0000] {logging_mixin.py:188} INFO - [2024-03-09T03:05:15.555+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T03:05:15.559+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:05:15.574+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.033 seconds
[2024-03-09T03:05:45.738+0000] {processor.py:161} INFO - Started process (PID=1455) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:05:45.739+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T03:05:45.740+0000] {logging_mixin.py:188} INFO - [2024-03-09T03:05:45.740+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:05:45.756+0000] {logging_mixin.py:188} INFO - [2024-03-09T03:05:45.752+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T03:05:45.758+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:05:45.777+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.044 seconds
[2024-03-09T03:06:15.934+0000] {processor.py:161} INFO - Started process (PID=1464) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:06:15.935+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T03:06:15.936+0000] {logging_mixin.py:188} INFO - [2024-03-09T03:06:15.936+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:06:15.951+0000] {logging_mixin.py:188} INFO - [2024-03-09T03:06:15.947+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T03:06:15.953+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:06:15.970+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.038 seconds
[2024-03-09T03:06:46.117+0000] {processor.py:161} INFO - Started process (PID=1473) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:06:46.118+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T03:06:46.119+0000] {logging_mixin.py:188} INFO - [2024-03-09T03:06:46.119+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:06:46.133+0000] {logging_mixin.py:188} INFO - [2024-03-09T03:06:46.129+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T03:06:46.135+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:06:46.149+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.035 seconds
[2024-03-09T03:07:16.293+0000] {processor.py:161} INFO - Started process (PID=1482) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:07:16.294+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T03:07:16.295+0000] {logging_mixin.py:188} INFO - [2024-03-09T03:07:16.295+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:07:16.307+0000] {logging_mixin.py:188} INFO - [2024-03-09T03:07:16.304+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T03:07:16.308+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:07:16.324+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.032 seconds
[2024-03-09T03:07:46.467+0000] {processor.py:161} INFO - Started process (PID=1491) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:07:46.468+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T03:07:46.468+0000] {logging_mixin.py:188} INFO - [2024-03-09T03:07:46.468+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:07:46.481+0000] {logging_mixin.py:188} INFO - [2024-03-09T03:07:46.478+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T03:07:46.482+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:07:46.498+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.033 seconds
[2024-03-09T03:08:16.645+0000] {processor.py:161} INFO - Started process (PID=1500) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:08:16.646+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T03:08:16.647+0000] {logging_mixin.py:188} INFO - [2024-03-09T03:08:16.646+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:08:16.660+0000] {logging_mixin.py:188} INFO - [2024-03-09T03:08:16.657+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T03:08:16.661+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:08:16.676+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.033 seconds
[2024-03-09T03:08:46.821+0000] {processor.py:161} INFO - Started process (PID=1509) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:08:46.822+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T03:08:46.822+0000] {logging_mixin.py:188} INFO - [2024-03-09T03:08:46.822+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:08:46.835+0000] {logging_mixin.py:188} INFO - [2024-03-09T03:08:46.832+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T03:08:46.836+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:08:46.851+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.033 seconds
[2024-03-09T03:09:16.992+0000] {processor.py:161} INFO - Started process (PID=1518) to work on /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:09:16.993+0000] {processor.py:830} INFO - Processing file /opt/airflow/dags/scrape_bikroy_dag.py for tasks to queue
[2024-03-09T03:09:16.994+0000] {logging_mixin.py:188} INFO - [2024-03-09T03:09:16.993+0000] {dagbag.py:540} INFO - Filling up the DagBag from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:09:17.007+0000] {logging_mixin.py:188} INFO - [2024-03-09T03:09:17.004+0000] {dagbag.py:350} ERROR - Failed to import: /opt/airflow/dags/scrape_bikroy_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.8/site-packages/airflow/models/dagbag.py", line 346, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/airflow/dags/scrape_bikroy_dag.py", line 4, in <module>
    import scrape_data_from_bikroy
  File "/opt/airflow/dags/scrape_data_from_bikroy.py", line 1, in <module>
    from selenium import webdriver
ModuleNotFoundError: No module named 'selenium'
[2024-03-09T03:09:17.008+0000] {processor.py:842} WARNING - No viable dags retrieved from /opt/airflow/dags/scrape_bikroy_dag.py
[2024-03-09T03:09:17.024+0000] {processor.py:183} INFO - Processing /opt/airflow/dags/scrape_bikroy_dag.py took 0.033 seconds
